### ISSUE TITLE
Better error message if PM2 is misconfigured / throwing EACCES error messages

### ISSUE THREAD

**Issue by kevinburke** [Unitech/pm2#3046]
If you configure/install PM2 as a root user, and then attempt to run any commands (help, version) as a non-root user, you get a cryptic error message:

```
$ pm2 --version
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: connect EACCES /home/ubuntu/.pm2/rpc.sock
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)
```

This is confusing. It would be better if pm2 could catch the error and report something like "Permission denied, double check you have not installed pm2 as root" or something.

**Comment by Unitech** [comment]
good idea

**Comment by Unitech** [comment]
Fixed on recently published pm2 (2.7.0):

```bash
$ npm install pm2@latest -g
$ pm2 update
```

Let me know if there is any issue
